plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -2,11 +2,13 @@
 {% set maxPagesDisplayedByVisitor=100 %}
 
 <ul id='visitsLive'>
-    {% for visitor in visitors %}
+    {% for visitor in dataTable.getRows() %}
+        {% set idSite = visitor.idSite %}
```

Comment:
Is the idSite variable used somewhere? Also is there a difference between `visitors` and `datatable.getRows()`?